### PR TITLE
Rename reserved "object" to "phpObject"; fixes #723

### LINKED ISSUE #723: Cannot use 'object' as class name as it is reserved
Hello,

Since today, I get failures on travis builds on php nightly:

```
==> Error FATAL ERROR in /home/travis/build/glpi-project/glpi/tests/units/Item_Devicesimcard.php on unknown line, generated by file /home/travis/build/glpi-project/glpi/vendor/atoum/atoum/classes/asserters/object.php on line 7:

Cannot use 'object' as class name as it is reserved
```

The PHP version is:
PHP 7.2.0-dev (cli) (built: Jun 29 2017 16:29:00) ( ZTS )

### PR DESCRIPTION
**Pull request by trasher**
This is a first try to get that fixed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * [#709](https://github.com/atoum/atoum/pull/709) Exception asserter now has `isInstanceOf` without parenthesis ([@guiled])
 * [#705](https://github.com/atoum/atoum/pull/705) Stream asserter now has isRead and isWritten assertion (without brackets) ([@guiled])
 * [#701](https://github.com/atoum/atoum/pull/701) Mock generator supports `strict_types` ([@jubianchi])
+* [#723](https://github.com/atoum/atoum/pull/724) "object" is a reserved keyword as of PHP 7.2 ([@trasher])
```
Comment (Hywan) [inline CHANGELOG.md]:
```diff
- "object"
+ `object`
```

**Comment by Hywan** [comment]
Thanks for the proposal! This is great. I have requested a very small change.

File: CHANGELOG.md (hunk 1 above)
Comment (trasher) [inline CHANGELOG.md]: Fixed, thanks

**Comment by Hywan** [comment]
The review from @jubianchi is mandatory here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 * [#709](https://github.com/atoum/atoum/pull/709) Exception asserter now has `isInstanceOf` without parenthesis ([@guiled])
 * [#705](https://github.com/atoum/atoum/pull/705) Stream asserter now has isRead and isWritten assertion (without brackets) ([@guiled])
 * [#701](https://github.com/atoum/atoum/pull/701) Mock generator supports `strict_types` ([@jubianchi])
+* [#723](https://github.com/atoum/atoum/pull/724) `object` is a reserved keyword as of PHP 7.2 ([@trasher])
```
Comment (jubianchi) [inline CHANGELOG.md]: this line should be the first in the changelog for the `dev-master` branch.

**Comment by trasher** [comment]
@jubianchi I'm happy if I can help  bit ;)

**Pull request merged**